pkgs/development/python-modules/pymumble/default.nix

```diff
@@ -1,31 +1,48 @@
 { buildPythonPackage,
   fetchFromGitHub,
+  fetchpatch,
   isPy27,
   lib,
   opuslib,
   protobuf,
+  pytest,
+  pycrypto,
 }:
 
 buildPythonPackage rec {
   pname = "pymumble";
-  version = "0.3.1";
+  version = "1.5";
   disabled = isPy27;
 
   src = fetchFromGitHub {
     owner = "azlux";
     repo = "pymumble";
-    rev = "1dd6d6d4df2fdef33202f17e2acf3ba9678a5737";
-    sha256 = "1r1sch8xrpbzffsb72lhp5xjr3ac3xb599n44vsfmaam3xklz6vz";
+    rev = "1.5";
+    sha256 = "0583j7vpjn3zvzcq1ga84608772m6zj0y3ika1kwrhy4l04hlx2n";
   };
+  patches = [
+    # Compatibility with pycryptodome (which is what our pycrypto really is)
+    # See https://github.com/azlux/pymumble/pull/99
+    (fetchpatch {
+      url = "https://github.com/azlux/pymumble/pull/99/commits/b85548a0e1deaac820954b1c0b308af214311a14.patch";
+      sha256 = "0w9dpc87rny6vmhi634pih1p97b67jm26qajscpa9wp6nphdlxlj";
+    })
+  ];
+
+  postPatch = ''
+    sed -i 's/\(.*\)==.*/\1/' requirements.txt
```

Comment:
This relaxes the version requirements, right?

Reply:
Indeed, transforms `foo=1.0` into just `foo`

Reply:
I think we might want to comment that.